### PR TITLE
Sort and Search activity fixes

### PR DESCRIPTION
**Pull request by sdwowk**

### CODE REVIEW

File: QuestionApp/.settings/org.eclipse.jdt.core.prefs [hunk 1]
```diff
@@ -1,6 +1,13 @@
 eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
 org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.6
+org.eclipse.jdt.core.compiler.codegen.unusedLocal=preserve
 org.eclipse.jdt.core.compiler.compliance=1.6
+org.eclipse.jdt.core.compiler.debug.lineNumber=generate
+org.eclipse.jdt.core.compiler.debug.localVariable=generate
```
Comment (RyanKusters) [inline QuestionApp/.settings/org.eclipse.jdt.core.prefs:7]: what is this?

File: QuestionApp/.settings/org.eclipse.jdt.core.prefs [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
+org.eclipse.jdt.core.compiler.debug.sourceFile=generate
```
Comment (linster) [inline QuestionApp/.settings/org.eclipse.jdt.core.prefs:8]: Can has CMPUT 301 project generator?

**Comment by stephenjust** [comment]
Good enough. Lets hope the project settings change doesn't make things blow up.

**Comment by RyanKusters** [comment]
looks good

**Pull request merged**